opentelemetry-zipkin/src/exporter/env.rs

```diff
@@ -0,0 +1,66 @@
+use std::env;
+use crate::ZipkinPipelineBuilder;
+
+/// HTTP endpoint for Zipkin collector.
+/// e.g. "http://localhost:9411/api/v2/spans"
+#[cfg(any(
+    feature = "reqwest-client",
+    feature = "surf-client",
+    feature = "reqwest-blocking-client"
+))]
+const ENV_ENDPOINT: &str = "OTEL_EXPORTER_ZIPKIN_ENDPOINT";
+
+/// Maximum time the Zipkin exporter will wait for each batch export
+#[cfg(any(
+    feature = "reqwest-client",
+    feature = "surf-client",
+    feature = "reqwest-blocking-client"
+))]
+const ENV_TIMEOUT: &str = "OTEL_EXPORTER_ZIPKIN_TIMEOUT";
+
+/// Assign builder attributes from env
+pub(crate) fn assign_attrs(mut builder: ZipkinPipelineBuilder) -> ZipkinPipelineBuilder {
+
+    #[cfg(any(
+        feature = "reqwest-client",
+        feature = "surf-client",
+        feature = "reqwest-blocking-client"
+    ))]
+    {
+        if let Some(endpoint) = env::var(ENV_ENDPOINT).ok().filter(|var| !var.is_empty()) {
+            builder = builder.with_collector_endpoint(endpoint);
+        }
+    }
+
+    #[cfg(any(
+        feature = "reqwest-client",
+        feature = "surf-client",
+        feature = "reqwest-blocking-client"
+    ))]
+    {
+        if let Some(timeout) = env::var(ENV_TIMEOUT).ok().filter(|var| !var.is_empty()) {
+            let timeout = duration_str::parse(&timeout)
```

Comment:
I don't think we need to parse duration here. [As defined in spec](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md#duration), all duration should be an integer representing duration in ms

Reply:
Indeed I missed that part.

Reply:
Also apparently should only warn and fail gracefully . I'll patch for that also

Reply:
Thanks a lot!